cfme/web_ui/menu.py

```diff
@@ -121,8 +142,8 @@ def get_rid_of_the_menu_box():
     ),
     ('infrastructure', 'Infrastructure'): (
```

Comment:
> I'd like to have you opinion on this file's changes. I introduced callables so we can decide what string to use at runtime.

My first impression was that using `callable` here is a good idea, and since it has helped us out almost everywhere else that we've implemented it, I don't see a reason not to support it.

I haven't taken a very close look beyond that, though.